tabs/tab_logit.py

```
@@ -311,25 +319,82 @@ def _update_inputs():
+             outcome_cols = [c for c in cols if pd.api.types.is_numeric_dtype(d[c])]
+
         # Identify potential subgroups (2-10 levels)
         sg_cols = [c for c in cols if 2 <= d[c].nunique() <= 10]
         
         # Update Tab 1 Inputs
-        ui.update_select("sel_outcome", choices=binary_cols)
+        ui.update_select("sel_outcome", choices=outcome_cols)
         ui.update_selectize("sel_exclude", choices=cols)
         
+        # Update Interaction Inputs
+        ui.update_select("int_var1", choices=[""] + cols)
```

Ruff (0.14.10)

Consider `["", *cols]` instead of concatenation

Replace with `["", *cols]`

(RUF005)